ember-rails.gemspec

```diff
@@ -15,6 +15,7 @@ Gem::Specification.new do |s|
   s.add_dependency "railties", [">= 3.1"]
   s.add_dependency "active_model_serializers"
   s.add_dependency "barber", [">= 0.6.0"]
+  s.add_dependency "es6_module_transpiler-rails"
```

Comment:
@tricknotes - esperanto is just for module transpilation, but 6to5 supports a large number of other ES6 (ES2015?) features.